Volume/Views/MainView/HomeList.swift

```diff
@@ -11,7 +11,9 @@ import SwiftUI
 
 struct HomeList: View {
     @State private var cancellableQuery: AnyCancellable?
+    @State private var openedUrl: Bool = false
     @State private var state: MainView.TabState<Results> = .loading
+    @State private var urlArticle: Article? = nil
```

Comment:
I don't understand what this variable means. Is there a better name? Maybe a doc comment?

Reply:
Would onOpenedArticleUrl work or is that too long?